data/json/items/ammo/40mm.json

```diff
@@ -107,5 +107,15 @@
     "description": "A 40mm grenade with a teargas load.  It will burst in a cloud of highly incapacitating gas.",
     "drop": "gasbomb_act",
     "extend": { "effects": [ "NO_EMBED" ] }
+  },
+  {
+    "id": "40mm_slug",
```

Comment:
Have a source for this? 40mm rounds are notorious for requiring a complex propellant mix to manage recoil, I would only imagine a slug is strictly worse.

Reply:
I agree, that's why I gave it a much higher recoil value.

Reply:
If you think the recoil is still insufficient I can raise it.

Reply:
On second thought, you can always sabot and shape your slug to reduce weight, so the question is how much slug and how much powder you can put in there.
The problem with trying to do reloads of the standard rounds is that your payload is fixed, but that doesn't apply here.

I'd still like to figure out what kind of load we're talking about here.

Reply:
After some offline discussion I *think* we reached a consensus that:
Damage is fine, this is a monster round.
Recoil might be ok to drop a bit.
The recipe needs something to represent a sabot.
Probably drop the lead and gunpowder in the recipe.